VVResonances/interactive/makeInputs.py

```diff
@@ -43,11 +51,16 @@
 
 purities=['HPHP','HPLP','LPLP','NP']
 purities=['HPHP','HPLP']
+purities=['HPHP']
 
-BulkGravWWTemplate="BulkGravToWW_narrow"
+BulkGravWWTemplate="BulkWW_"
 BulkGravZZTemplate="BulkGravToZZToZhadZhad_narrow"
+WprimeTemplate= "WprimeToWZ"
+ZprimeWWTemplate= "ZprimeWW"
+# use arbitrary cross section 0.001 so limits converge better
 BRWW=1.*0.001
-BRZZ=1.*0.001
+BRZZ=1.*0.001*0.6991*0.6991
```

Comment:
We need to make some changes here to be able to run all signals with same script